### DISCUSSION TITLE
hiển thị thay đổi trong sơ đồ thống kê

### DISCUSSION THREAD

**Discussion by vdt2210** [trvqhuy/ha-evn#48] [Q&A]
trước khi update bản 2.4.1 mình có setting như vậy thì sơ đồ hiển thị giá trị đã thay đổi so vs hqua (h2)
![image](https://github.com/trvqhuy/nestup_evn/assets/87288753/edc0c7e3-af79-4e92-b09f-fc3628afa3bd)
thì sơ đồ hiển thị giá trị đã thay đổi so vs hqua
![image](https://github.com/trvqhuy/nestup_evn/assets/87288753/5863a5e5-7f41-4d32-a74e-3e4d58dfc69b)

nhưng sau khi update thì giá trị bị đổi thành số điện tạm chốt luôn
![image](https://github.com/trvqhuy/nestup_evn/assets/87288753/e2887dab-495a-4464-be69-4d80c7600a8d)

ko biết này là do lỗi hay sao vậy?

EDIT:
bên phần energy cũng bị như trên luôn
![image](https://github.com/trvqhuy/nestup_evn/assets/87288753/4b705775-28ac-48c4-940b-59c22d676d98)
![image](https://github.com/trvqhuy/nestup_evn/assets/87288753/adf66a02-504c-4711-bcf7-54af872290ed)

**Comment by trvqhuy** [comment]
Bạn thử setup lại Energy Dashboard Entity giúp mình nhé.

**Reply by vdt2210** [reply]
mình setup lại vẫn hiện chỉ số tổng tạm chốt à bạn

**Comment by vdt2210** [comment]
@trvqhuy  chart nay bình thường trờ lại rồi. nhưng thắc mắc tí ko biết số "thay đổi" trong hình lấy như nào nhỉ? mình xem trên app của evn thì lệch 2 ngày
![Screenshot_20240616-134312](https://github.com/trvqhuy/nestup_evn/assets/87288753/fe13bfc4-96d8-45a9-84af-1ae592cfd8f5)
![image](https://github.com/trvqhuy/nestup_evn/assets/87288753/3047711d-322b-4aed-8100-68d230285a46)
